python/ray/actor.py

```diff
@@ -590,6 +593,12 @@ def _remote(self,
                 this actor or task and its children (see
                 :ref:`runtime-environments` for details).  This API is in beta
                 and may change before becoming stable.
+            max_pending_calls (int): Set the maximum number of pending calls
```

Comment:
cc @ericl @scv119  can you review the API doc?

Reply:
Btw, we should mention what's the default value mean (-1)

Reply:
Please check the grammar on all documentation. Here is a rewritten version:
```
max_pending_calls (int): Set the max number of pending calls allowed on the actor handle.
When this value is exceeded, PendingCallsLimitExceeded will be raised for further tasks.
Note that this limit is counted per handle.
```

Reply:
default value meaning added

Reply:
@ericl Documentation and some logs refactored, please have a look.